### PR TITLE
Adding xaxxon_openlidar to documentation index for distro

### PR DESCRIPTION
**Pull request by xaxxontech**

### CODE REVIEW

**Review by tfoote — CHANGES_REQUESTED** [review]
This is failing CI as the url's don't end in .git Please add that to the urls.

**Comment by xaxxontech** [comment]
thanks - '.git's added!

File: indigo/distribution.yaml [hunk 1]
```diff
@@ -17769,6 +17769,16 @@ repositories:
       url: https://github.com/ros/xacro.git
       version: indigo-devel
     status: maintained
+  xaxxon_openlidar:
```
Comment (sloretz) [inline indigo/distribution.yaml]: @tfoote since Indigo and Lunar are EOL, should the new entries in these two distros be removed from this PR?

Comment (tfoote) [reply indigo/distribution.yaml]: Yes please only update kinetic and melodic due to the others being EOL.

**Pull request merged**